migrations/versions/847b0850ea81_drop_sha_and_repository.py

```diff
@@ -0,0 +1,60 @@
+"""Drop sha and repository
+
+Revision ID: 847b0850ea81
+Revises: c181484ce40f
+Create Date: 2021-07-29 11:16:43.377407
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "847b0850ea81"
+down_revision = "c181484ce40f"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.alter_column(
+        "distribution", "commit_id", existing_type=sa.VARCHAR(length=50), nullable=False
+    )
+    op.drop_index("distribution_repository_index", table_name="distribution")
+    op.drop_index("distribution_sha_index", table_name="distribution")
+    op.drop_index("distribution_index", table_name="distribution")
+    op.create_index(
+        "distribution_index",
+        "distribution",
+        ["case_id", "context_id", "commit_id", "machine_hash"],
+        unique=True,
```

Comment:
@dianaclarke will you please also drop `distribution_case_id_index` since composite index `distribution_index` can be used by Postgres to do the same?